Automattic/jetpack
# Add Boost and CRM Cards to AAG

#### Changes proposed in this Pull Request:


- Add new `DashPluginItem` component for managing separate Plugins as cards in the Jetpack AAG Dashboard
- Use `DashPluginItem` to add new CRM and Boost Card to the AAG Dashboard
- Rename "Performance" section to "Performance and Growth"

##### `DashPluginItem` 

| State | Screenshot |
| -------|--------------------------|
| Loading | <img width="477" alt="Screen Shot 2021-12-15 at 15 44 42" src="https://user-images.githubusercontent.com/2810519/146298950-8d9a0d46-0e1d-462a-8ad5-ef6e5ed4b73a.png"> |
| Prompt Install/Activate Plugin | <img width="479" alt="Screen Shot 2021-12-15 at 18 43 34" src="https://user-images.githubusercontent.com/2810519/146299016-f7f43409-f05c-42ea-8010-0c6ba4f2ee8b.png"> |
| Installing/Activating Plugin | <img width="476" alt="Screen Shot 2021-12-15 at 15 46 08" src="https://user-images.githubusercontent.com/2810519/146299113-e4a3b644-634d-4b60-93c4-3734ec9e1dfa.png"> |
| Manage Plugin | <img width="491" alt="Screen Shot 2021-12-15 at 15 38 05" src="https://user-images.githubusercontent.com/2810519/146299390-2f8ca82c-2f08-4e9c-b055-d1c27ed1343e.png"> |

##### Overal Changes

<img width="982" alt="Screen Shot 2021-12-15 at 18 46 24" src="https://user-images.githubusercontent.com/2810519/146299477-4653348d-86da-4d94-86bb-ce71ed4ae1fb.png">

##### Video Demonstration

https://user-images.githubusercontent.com/2810519/146299579-1f8916db-e17a-470c-bdde-3c4f492ebcab.mov

#### Jetpack product discussion



p1HpG7-dHF-p2

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Navigate to `/wp-admin/admin.php?page=jetpack#/dashboard` on site with branch
2. Test each of the following with both the CRM and Boost Plugins
 a. When the Plugin is not installed it shows the "install" prompt.
 b. Clicking "Install Plugin" correctly installs the plugin.
 c. When the plugin is installed but not activated the "Activate Plugin" prompt is shown.
 d. Clicking "Activate Plugin" correctly activates the plugin.
 e. When the Plugin is installed and activated the "Manage Plugin" prompt is shown
 f. Clicking the "Manage Plugin" plugin navigates to the respective plugin dashboard
